avaframe/data/avaSimilaritySol/Inputs/simiSol_com1DFACfg.ini

```diff
@@ -103,4 +106,4 @@ dtSol = 50.
 
 # TODO still here user interaction
 # user interaction for plots
-flagInteraction = False
+flagInteraction = True
```

Comment:
should be default False

Reply:
How do we do this, because here there is no local so we have every time t change it ourselves...